scripts/benchmark_db/bench.rb

```diff
@@ -4,27 +4,48 @@
 
 # Script to test various configurations that can impact performance of the node
 
+require 'concurrent-ruby'
+require 'csv'
 require 'deep_merge'
 require 'fileutils'
 require 'open3'
 require 'optparse'
-require 'pathname'
 require 'set'
 require 'tmpdir'
+require 'tomlrb'
 require 'toml-rb'
```

Comment:
Do we need both?

Reply:
Yes. Unfortunately, one doesn't support correct parsing of toml spec. And we need the other for some functionality.